Src/ILGPU/Frontend/Intrinsic/RemappedIntrinsics.Generated.tt

```diff
@@ -96,16 +96,16 @@ namespace ILGPU.Frontend.Intrinsic
 #if NETCORE
             var systemMathFType = typeof(MathF);
 
-<# foreach (var (method, type, _, className) in #>
-<#      unaryMathFunctions.Where(t => t.Item2 != null)) { #>
+<# foreach (var (method, type, _, className) in unaryMathFunctions
```

Comment:
This is interesting; it looks like the T4-based parser may fail when using the previous code.

Reply:
Yes the template wouldn't build for me without this change, I wasn't sure whether that was due to a difference in Visual Studio versions or whether it was broken for everyone but hadn't been noticed.

Reply:
I've spend some time investigating these issues. It seems to be a problem with different SDK versions.

Reply:
I have encountered similar problems in the past regarding the "linefeed" characters appended to the end of the file, which had to be removed to have build support for Linux.